packages/portal/src/graphql/queries/landingPage.graphql

```diff
@@ -106,20 +114,7 @@ query LandingPage(
     }
   }
 }
-fragment imageFields on Asset {
-  title
-  url
-  contentType
-  description
-  width
-  height
-  contentfulMetadata {
```

Comment:
This was the only place we are getting contentfulMetadata and the asset title back.
I don't think they're being used, but since they're included in the new fragment it's going to add unnecessary object data to any page/component that's using contentful images.